core/src/main/java/org/apache/accumulo/core/clientImpl/mapreduce/lib/ConfiguratorBase.java

```diff
@@ -62,6 +63,8 @@
     IS_CONFIGURED, PRINCIPAL, TOKEN
   }
 
+  public static final String tokenFileName = "tokenfile";
```

Comment:
This represents the URI fragment after `#` which is to be used for the local file name in the distributed cache. So, the variable name could be something like "localFileName" or "cachedFileName" or similar.